components/scream/src/physics/rrtmgp/share/shr_orb_mod_c.hpp

```diff
@@ -0,0 +1,14 @@
+#ifndef SHR_ORB_MOD_C_HPP
+#define SHR_ORB_MOD_C_HPP
+extern "C" void shr_orb_params_c(
```

Comment:
I still think our "_c" and "_f90" convention is backwards.

Reply:
Apologies, I’ll fix.

Reply:
Okay I think I fixed this. I renamed the bridges with _c2f since it creates a C interface that calls F90 code. Let me know if this isn’t quite what you were looking for.